### ISSUE TITLE
Key Derivation not Working

### ISSUE THREAD

**Issue by filippoweb3** [Kalapaja/siltti#8]
I tried to use the key Derviation option, but it seems is not working properly.

![CF5F0F3E-1999-4870-A765-EFD192FCA88E_1_201_a](https://github.com/Kalapaja/siltti/assets/110459737/69773051-0768-415c-adbe-cdf51ddfce52)

I was expecting I could send the derivation path payload to the signer to get the derived address, but it seems the QR code always shoes the parent address.

**Comment by Slesarew** [comment]
Yes, it's not yet implemented on Kampela side. Coming soon (the code is in https://github.com/Kalapaja/kampela-firmware/pull/18 but still need to do modifications there to comply with security model. The security model itself is in process of being reworked due to https://github.com/Alzymologist/RFCs/blob/metadata-for-offline-signers/text/0046-metadata-for-offline-signers.md that changes a lot. Soon upgrades waiting in line should start shipping one after another.)
